### ISSUE TITLE
Formly-form mangled when viewed from a favourites or trending page

### ISSUE THREAD

**Issue by garfield69** [SchizoDuckie/DuckieTV#942]
**What build of DuckieTV are you using (Standlone / Chrome Extension (New Tab / Browser Action))**
Standalone and Chrome Extension
**What version of DuckieTV are you using (Stable 1.1.x / Nightly yyyymmddHHMM)**
1.1.4 or latest Nightly

**What is your Operating System (Windows, Mac, Linux, Android)**
windows

**Describe the problem you are having and steps to reproduce if available**
the formly-form for serieSettings seems to get mangled once the favourites or trending pages are accessed.     
The forms present correctly while on the calendar page, but once a favourites or trending page is viewed, the forms are mangled from then on, until the next reset.

example: 
- start up DuckieTV and use fastsearch or the calendar to pull up a series and access the seriesSettings
![image](https://user-images.githubusercontent.com/6933240/29556918-7bf1dfb2-877b-11e7-858a-bb9e35a3a84b.png)

- now access the same series from the favourites view and access the seriesSettings
![image](https://user-images.githubusercontent.com/6933240/29556976-b89f585e-877b-11e7-8e14-44e530d7af2c.png)

**Attach any DuckieTV statistics or Developer Console logs if available**
...

**Comment by garfield69** [comment]
I really have no idea what is going on here.       

How does viewing the favourites or trending pages alter the environment such that the formly-form dialogue gets mangled in such as strange way?        

Why does the *display specials* checkbox appear below the *auto download* checkbox, when the *later* is defined *after* the *former* in *serieSettings.json*?      
The two fields are meant to be on one row.... has the row limits shrunk and the fields no longer fit?      

Where do I even start looking to begin diagnosing the problem? The *inspect* panel on the *development tools* is full of information, so much in fact that I am getting info overload.      

Help with this would be very welcome, anyone, please? Thanks.

**Comment by Js41637** [comment]
I can reproduce this issue.

**Comment by Js41637** [comment]
I cannot reproduce this issue.

**Comment by garfield69** [comment]
@Js41637  OK, that is interesting.      
I'm going to spell out the steps in more details which I would like you to try out please.     
1. click on heart icon 
![image](https://user-images.githubusercontent.com/6933240/29680913-2a44e49e-895a-11e7-805c-98d9a58e251e.png)
2. click on a poster
![image](https://user-images.githubusercontent.com/6933240/29680951-4d83a86e-895a-11e7-8d9d-70721464b96a.png)
3. click on the Series Settings button
![image](https://user-images.githubusercontent.com/6933240/29680991-74380342-895a-11e7-96b5-5b7dfc281e13.png)
4 confirm you get the mangled dialogue
![image](https://user-images.githubusercontent.com/6933240/29681046-a3b0fcb4-895a-11e7-885e-e7cc341ec0e8.png)

If the dialogue looks ok to you, then I wonder if it has something to do with x64 versus x32 ?      
All my platforms are x32, and I can get this problem even on my Ubuntu sandbox.      

Just in case it is a settings issue, could you:
1. click on the about icon
![image](https://user-images.githubusercontent.com/6933240/29681257-4020b5c6-895b-11e7-94cb-f17266be84d2.png)
2. click on the statistics header
![image](https://user-images.githubusercontent.com/6933240/29681290-5cc008bc-895b-11e7-8feb-b5060e78bd4e.png)
3. and paste below. I'd like to compare settings and see if there is something different about your setup.      

As a comparison, here is my statistics dump, I've redacted the unnecessary stuff to condense the output.
```javascript
[
  {    "name": "DuckieTV Web Based",    "data": "1.1.5"  },
  {    "name": "Current Date and Time",    "data": "Thu, 24 Aug 2017 18:05:21 GMT"  },
  {    "name": "NWJS, Chromium",    "data": "0.24.3 , 60.0.3112.101"  },
  {    "name": "UserAgent",    "data": "DuckieTV-Standalone-Development v1.1.5 Windows NT 6.1"  },
  {    "name": "Platform, Vendor",    "data": "Win32, Google Inc."  },
  {    "name": "Screen (width x height)",    "data": "1536 x 864"  },
  {    "name": "Default Search Engine",    "data": "ThePirateBay [Jackett] (Enabled)"  },
  {    "name": "Default Torrent Client",    "data": "uTorrent Web UI (Enabled but Not Connected to Host)"  },
  {    "name": "Auto Download Service",    "data": "(Enabled and Active)"  },
  {    "name": "Last checked TraktTV for DB updates on",    "data": "Thu, 24 Aug 2017 17:56:37 GMT"  },
  {    "name": "User Preferences on Local Storage",
    "data": "{\n  \"ThePirateBay.mirror\": \"https://thepiratebay.org\",\n  \"application.language\": \"en_us\",\n  \"application.locale\": \"en_us\",\n  \"autobackup.period\": \"weekly\",\n  \"autodownload.delay\": 240,\n  \"autodownload.lastrun\": 1503570251190,\n  \"autodownload.minSeeders\": 20,\n  \"autodownload.period\": 7,\n  \"calendar.mode\": \"week\",\n  \"calendar.show-episode-numbers\": true,\n  \"calendar.show-specials\": false,\n  \"calendar.startSunday\": false,\n  \"client.determinedlocale\": \"en_us\",\n  \"deluge.password\": \"*****\",\n  \"deluge.port\": 8112,\n  \"deluge.server\": \"http://localhost\",\n  \"kc.always\": true,\n  \"ktorrent.password\": \"*****\",\n  \"ktorrent.port\": 8080,\n  \"ktorrent.server\": \"http://192.168.20.101\",\n  \"ktorrent.username\": \"garfield69\",\n  \"main.viewmode\": \"calendar\",\n  \"qbittorrent.password\": \"*****\",\n  \"qbittorrent.server\": \"http://localhost\",\n  \"qbittorrent32plus.password\": \"*****\",\n  \"qbittorrent32plus.port\": 8080,\n  \"qbittorrent32plus.server\": \"http://localhost\",\n  \"qbittorrent32plus.use_auth\": true,\n  \"qbittorrent32plus.username\": \"garfield69\",\n  \"rtorrent.path\": \"/RPC2\",\n  \"rtorrent.port\": 80,\n  \"rtorrent.server\": \"http://192.168.20.101\",\n  \"series.not-watched-eps-btn\": true,\n  \"synology.enabled\": false,\n  \"tixati.password\": \"*****\",\n  \"tixati.port\": 8888,\n  \"tixati.server\": \"http://localhost\",\n  \"tixati.use_auth\": true,\n  \"tixati.username\": \"admin\",\n  \"topSites.enabled\": false,\n  \"topSites.mode\": \"onclick\",\n  \"torrentDialog.2.enabled\": true,\n  \"torrentDialog.2.activeSE\": {\n    \"1337x\": true,\n    \"EzTV.ag\": true,\n    \"Idope\": true,\n    \"IsoHunt\": true,\n    \"KATcr\": false,\n    \"LimeTorrents\": true,\n    \"Nyaa\": false,\n    \"RarBG\": true,\n    \"ShowRSS\": false,\n    \"SkyTorrents\": true,\n    \"ThePirateBay\": true,\n    \"TorrentDownloads\": true,\n    \"TorrentZ2\": true,\n    \"Zooqle\": true,\n    \"Horrible Subs\": true,\n    \"TorrentProject\": true,\n    \"HorribleSubs\": false,\n    \"tpb\": true,\n    \"Jackett\": true,\n    \"RuTracker\": true,\n    \"Demonoid\": true,\n    \"YTSag\": false,\n    \"AllJackett\": false,\n    \"TorLock\": true,\n    \"Anidex\": false,\n    \"AnimeTosho\": false,\n    \"AniRena\": false\n  },\n  \"torrentDialog.showAdvanced.enabled\": true,\n  \"torrenting.autodownload\": true,\n  \"torrenting.autostop\": true,\n  \"torrenting.autostop_all\": true,\n  \"torrenting.client\": \"uTorrent Web UI\",\n  \"torrenting.enabled\": true,\n  \"torrenting.global_size_max\": null,\n  \"torrenting.global_size_max_enabled\": false,\n  \"torrenting.global_size_min\": null,\n  \"torrenting.global_size_min_enabled\": false,\n  \"torrenting.ignore_keywords\": \"xvid 480p dvdscr camrip preair trailer hdts hdtc 10bit\",\n  \"torrenting.ignore_keywords_enabled\": false,\n  \"torrenting.label\": true,\n  \"torrenting.launch_via_chromium\": false,\n  \"torrenting.min_seeders\": 20,\n  \"torrenting.min_seeders_enabled\": false,\n  \"torrenting.progress\": true,\n  \"torrenting.require_keywords\": \"rmteam shadow megusta\",\n  \"torrenting.require_keywords_enabled\": false,\n  \"torrenting.searchquality\": \"720p x265\",\n  \"torrenting.searchqualitylist\": [\n    \"720p\",\n    \"1080p\",\n    \"1080i\",\n    \"720p x264\",\n    \"720p x265\",\n    \"1080p x264\",\n    \"1080p x265\",\n    \"1080p DTS\",\n    \"x264\",\n    \"x265\"\n  ],\n  \"torrenting.streaming\": false,\n  \"trakt-update.period\": 2,\n  \"trakttv.sync\": true,\n  \"transmission.password\": \"*****\",\n  \"transmission.path\": \"/transmission/rpc\",\n  \"transmission.port\": 9091,\n  \"transmission.server\": \"http://localhost\",\n  \"transmission.use_auth\": true,\n  \"transmission.username\": \"garfield69\",\n  \"utorrentwebui.password\": \"*****\",\n  \"utorrentwebui.port\": 8080,\n  \"utorrentwebui.server\": \"http://localhost\",\n  \"utorrentwebui.use_auth\": true,\n  \"utorrentwebui.username\": \"garfield69\",\n  \"vuze.password\": \"*****\",\n  \"vuze.path\": \"/transmission/rpc\",\n  \"vuze.port\": 9091,\n  \"vuze.server\": \"http://localhost\",\n  \"vuze.use_auth\": true,\n  \"vuze.username\": \"garfield69\",\n  \"deluge.use_auth\": true,\n  \"torrentDialog.2.sortBy\": \"-engine\",\n  \"library.order.by\": \"getSortName()\",\n  \"library.order.reverseList\": [\n    true,\n    false,\n    true,\n    true\n  ],\n  \"qbittorrent.port\": 8080,\n  \"qbittorrent.username\": \"garfield69\",\n  \"qbittorrent.use_auth\": true,\n  \"autodownload.multiSE.enabled\": false,\n  \"autodownload.multiSE\": {\n    \"ThePirateBay\": true,\n    \"1337x\": false,\n    \"IsoHunt\": false,\n    \"KATcr\": false,\n    \"Nyaa\": false,\n    \"RarBG\": false,\n    \"ShowRSS\": false,\n    \"TorrentDownloads\": false,\n    \"TorrentZ2\": false,\n    \"Zooqle\": false,\n    \"EzTV.ag\": false,\n    \"Idope\": false,\n    \"LimeTorrents\": false,\n    \"SkyTorrents\": false,\n    \"HorribleSubs\": false,\n    \"TorrentProject\": false,\n    \"RuTracker\": false,\n    \"Demonoid\": false,\n    \"YTSag\": false,\n    \"AllJackett\": true,\n    \"TPB\": true,\n    \"TorLock\": true,\n    \"Anidex\": true,\n    \"AnimeTosho\": true,\n    \"AniRena\": true\n  },\n  \"biglybt.server\": \"http://localhost\",\n  \"biglybt.port\": 9091,\n  \"biglybt.username\": \"garfield69\",\n  \"biglybt.password\": \"*****\",\n  \"biglybt.use_auth\": true,\n  \"biglybt.path\": \"/transmission/rpc\",\n  \"torrenting.require_keywords_mode_or\": true\n}"
  },
  {    "name": "Other significant Local Storage keys",
    "data": "{\n  \"1.1.4TorrentHashListCleanup\": \"***redacted***\",\n  \"1.1.4cleanupOrphanedEpisodes\": \"***redacted***\",\n  \"1.1.4cleanupOrphanedSeasons\": \"***redacted***\",\n  \"1.1.4qBittorrentPre32\": \"***redacted***\": \"***redacted***\",\n  \"1.1.4refreshTrendingCache\": \"***redacted***\",\n  \"1.1.4updateDelugeAuth\": \"***redacted***\",\n  \"1.1.4updateTransmissionPath\": \"***redacted***\",\n  \"1.1.5FetchFirstXemAliasMap\": \"***redacted***\",\n  \"1.1.5deleteSearchEngines\": \"***redacted***\",\n  \"1.1.5fanartCleanup\": \"***redacted***\",\n  \"1.1.5updateDelugeAuth\": \"***redacted***\",\n  \"1.1.5updateTorrenting.min_seeders\": \"***redacted***\",\n  \"1.1migration\": \"***redacted***\",\n  \"CRUD.DEBUG\": \"false\",\n  \"autobackup.lastrun\": \"1503005320347\",\n  \"database.version.Episodes\": \"14\",\n  \"database.version.Fanart\": \"1\",\n  \"database.version.Jackett\": \"2\",\n  \"database.version.Seasons\": \"6\",\n  \"database.version.Series\": \"16\",\n  \"database.version.WatchList\": \"1\",\n  \"database.version.WatchListObject\": \"1\",\n  \"fanart.bootstrapped\": \"1\",\n  \"github.lastupdatecheck\": \"1443630662476\",\n  \"install.notify\": \"1.1.5\",\n  \"runtime.event\": \"{\\\"reason\\\":\\\"install\\\"}\",\n  \"snr.lastFetched\": \"1503550275741\",\n  \"standalone.alwaysShowTray\": \"Y\",\n  \"standalone.closeSystray\": \"N\",\n  \"standalone.minimizeSystray\": \"Y\",\n  \"standalone.position\": \"{\\\"width\\\":1538,\\\"height\\\":828,\\\"x\\\":0,\\\"y\\\":0,\\\"state\\\":\\\"maximized\\\"}\",\n  \"standalone.startupMinimized\": \"N\",\n  \"standalone.trayColor\": \"-inverted\",\n  \"standalone.zoomlevel\": \"6\",\n  \"torrenting.client\": \"uTorrent Web UI\",\n  \"trackers.fallBackList\": \"***redacted***",\n  \"trackers.lastFetched\": \"1501364853554\",\n  \"trakttv.lastupdated\": \"1503597397996\",\n  \"trakttv.lastupdated.trending\": \"1503570239052\",\n  \"trakttv.refresh_token\": \"***redacted***\",\n  \"uniqueId\": \"***redacted***\",\n  \"upgrade.notify\": \"1.1.5\",\n  \"utorrent.port\": \"10000\",\n  \"utorrent.token\": \"***redacted***\",\n  \"xem.aliasmap\": \"{***redacted***]}\",\n  \"xem.lastFetched\": \"1503570217386\"\n}"
  },
  {    "name": "DB Series",    "data": 256  },
  {    "name": "DB Series Hidden From Calendar",    "data": 0  },
  {    "name": "DB Seasons",    "data": 1123  },
  {    "name": "DB Episodes",    "data": 15429  },
  {    "name": "DB Fanart",    "data": 2336  },
  {    "name": "DB Jackett",    "data": 20  }
]
```

Thanks.

**Comment by Js41637** [comment]
Ok interesting. Forgot that when I said that I was testing using the Canary version of chrome. This issue occurs for me when using the latest stable v60. Which is really weird.

**Comment by Js41637** [comment]
I'll see if I can take another look.

**Comment by garfield69** [comment]
Excellent observation @Js41637 . I can confirm that switching to using Chrome Beta (M61) or Chrome Canary (M62) on my x32 platforms prevents the dialogue from getting mangled. So it is only happening on Chrome Stable (M60).      

So if we hold releasing DuckieTV 1.1.5 until after M61 becomes stable (currently Chrome 61 is estimated to release on September 12th) then we will be sweet, unless of course, you find a way to make the problem go away on M60 ;-)

**Comment by garfield69** [comment]
Confirming that this is fixed with Google Stable 61.0.3163.79 (Official Build) just released a moment ago.
